frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -17,6 +17,14 @@ type CandidateDeciderProps = {
 };
 
 const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isBypassingModal, setIsBypassingModal] = useState(false);
```

Comment:
I'm pretty sure we can get rid of this `isBypassingModal` entirely. It feels like a combination of `force` and `isSaved` is enough.